packages/core/src/libraries/hook/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { type HookEventPayload } from '@logto/schemas';
+import { conditional, trySafe } from '@silverhand/essentials';
+import { type OptionsOfTextResponseBody, type Response } from 'got';
+
+import { generateSignature } from '#src/utils/signature.js';
+
+export const parseResponse = ({ statusCode, body }: Response) => ({
```

Comment:
should it be a general util method?

Reply:
This method is only used in the hook logic, and I decided to keep it unmodified in this PR.
This code is reverted, and in the next pr, I will reorg it.